packages/loopring_v3/contracts/impl/SignatureBasedAddressWhitelist.sol

```diff
@@ -56,15 +58,18 @@ contract SignatureBasedAddressWhitelist is Claimable, IAddressWhitelist
             return false;
         }
 
+        if (uint256(s) > 0x7FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF5D576E7357A4501DDFE92F46681B20A0) {
+            return false;
+        }
+
         if (v < 27) {
             v += 27;
         }
-
         if (v != 27 && v != 28) {
             return false;
         }
-
-        bytes32 hash = keccak256(abi.encode("LOOPRING_DEX_ACCOUNT_CREATION", addr, t));
+        bytes32 msgBase = keccak256(abi.encodePacked("LOOPRING_DEX_ACCOUNT_CREATION", addr, t));
```

Comment:
"LOOPRING_DEX_ACCOUNT_CREATION"
another constant?

Reply:
Yes, I think it's better not to use a constant member variable, but use this kind of constant string.

Reply:
So this time I removed the const string member but use
`bytes32 hash = keccak256(abi.encodePacked("\x19Ethereum Signed Message:\n32", msgBase));`
instead.